packages/cli/generators/relation/templates/controller-relation-template-has-one.ts.ejs

```diff
@@ -0,0 +1,110 @@
+import {
+  Count,
+  CountSchema,
+  Filter,
+  repository,
+  Where,
+} from '@loopback/repository';
+import {
+  del,
+  get,
+  getModelSchemaRef,
+  getWhereSchemaFor,
+  param,
+  patch,
+  post,
+  requestBody,
+} from '@loopback/rest';
+import {
+  <%= sourceModelClassName %>,
+  <%= targetModelClassName %>,
+} from '../models';
+import {<%= sourceRepositoryClassName %>} from '../repositories';
+
+export class <%= controllerClassName %> {
+  constructor(
+    @repository(<%= sourceRepositoryClassName %>) protected <%= paramSourceRepository %>: <%= sourceRepositoryClassName %>,
+  ) { }
+
+  @get('/<%= sourceModelPath %>/{id}/<%= targetModelPath %>', {
```

Comment:
`targetModelPath` is currently plural, but we have a 1-to-1.

Reply:
Should I replace it with targetModelName or equate targetModelPath to targetModelName in has-one-relation.generator.ts? For now I have replaced it with targetModelName.

Reply:
I think I will also have to modify for patch, post and delete. So I have equated `targetModelPath` to `targetModelName` in `has-one-relation.generator.ts`. And I am using `targetModelPath` everywhere in the template so that others don't get confused as to why hasOne template is not consistent with the other templates. Let me know if I need to change anything. Thanks a lot for the quick response @agnes512 @derdeka